src/data.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+  // Used in ejs/includes/faq.ejs
+  faqs: [{
+    q: 'What is this?',
+    a: `
+      This is a service to allow you to buy cryptocurrency assets without the
+      need for an exchange in just a few minutes.
+    `
+  }, {
+    q: 'Who provides this service?',
+    a: `
+      The purchasing process is handled by our partners at Simplex.com. This
+      page is hosted by MyCrypto.
+    `
+  }, {
+    q: 'Are there any fees?',
+    a: 'There is a Simplex fee of 5% ($10 minimum) and a 2% tip to MyCrypto.'
```

Comment:
Up to 3%.

Reply:
Potentially make it clearer that the fee is 5% OR $10, whichever is higher.

Reply:
Ok cool. I'll submit a PR (I think that's the right term) for any changes this evening ideally.